components/raftstore/src/store/worker/pd.rs

```diff
@@ -726,6 +748,11 @@ where
             self.store_stat.engine_total_keys_read - self.store_stat.engine_last_total_keys_read,
         );
 
+        self.store_stat
+            .query_stats_since_last_report
+            .add_query_stats(stats.get_query_stats()); // add write query 
+        stats.set_query_stats(self.store_stat.query_stats_since_last_report.pop());
+
```

Comment:
This seems strange.

Reply:
I'll let it be written in line with the other stats, but I think it makes more sense to use the post-zero